### ISSUE TITLE
Minor road tunnels

### ISSUE THREAD

**Issue by danieldegroot2** [ZeLonewolf/openstreetmap-americana#1029]
Example: https://zelonewolf.github.io/openstreetmap-americana/#map=14.59/40.77207/-73.97353

On low zoom, these roads are all black instead of some colour like other major road types (freeway, etc.)
On high zoom, these roads are all white.
Tunnels, on the other hand, are white also on low zoom, creating a strong contrast between black roads and white tunnels.
This also makes the road type somewhat ambiguous.
Major road tunnels are subtly coloured to avoid this.

![image](https://github.com/ZeLonewolf/openstreetmap-americana/assets/67521919/f0eb07c4-003e-4b7b-870b-73d4a2d5d5e7)

**Comment by claysmalley** [comment]
I agree, it looks clunky. Fortunately, a rewrite that will substantially change tunnel rendering is in development: #995
